src/lib/bootloader/grub2_widgets.rb

```diff
@@ -435,16 +437,16 @@ def store
 
     def help
       _(
-        "<p><b>Protect Boot Loader with Password</b><br>\n" \
-          "At boot time, modifying or even booting any entry will require the" \
+        "<p><b>Protect Boot Loader with Password</b>\n" \
+          "at boot time, modifying or even booting any entry will require the" \
```

Comment:
This is inconsistent consistent here, the other lines use trailing space at the end of the line.

Reply:
it is not visible, so does not matter much for me. But I can add it.